src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -213,7 +213,7 @@ namespace plugins
     public: void TextureId(void* _texturePtr);
 
     /// \brief Render engine to use
-    public: std::string engineName = "ogre";
+    public: std::string engineName = "ogre2";
```

Comment:
maybe we should change the default in harmonic so this does not cause a behavior change?

Reply:
Agreed. gz-sim defaults to ogre2 through its default gui.config so no harm in waiting till Harmonic for this.

Reply:
This is now defaulting to "ogre" in this branch.